### PR TITLE
optimalvalues should reflect point of diminishing returns. 

### PR DESCRIPTION
**Pull request by paulirish**
some cleanup of the report as well: 
image

### CODE REVIEW

File: report/templates/report.html [hunk 1]
```diff
@@ -135,16 +137,18 @@ <h1 class="report-section__subtitle">{{ @key }}</h1>
                       <span class="report-section__item-description">
                         {{ subItem.description }}
                         {{#if subItem.optimalValue }}
-                        (target value: {{ subItem.optimalValue }} )
+                          <!--<small>-->
```
Comment (paullewis) [inline report/templates/report.html]: We should remove these.

**Comment by paullewis** [comment]
Super minor nits.

**Comment by brendankenny** [comment]
needs rebase but otherwise LGTM

**Comment by samccone** [comment]
+3

**Comment by paulirish** [comment]
rebased. 
merge when tests go green.

**Pull request merged**